Notebooks/FitMagProfile.ipynb

```diff
@@ -93,24 +93,52 @@
dh0JtqGhIbKz\ns+UZYpuGmeHgzZXuxYsXERAQUO8xBwcHZGRkQENDA2FhYfjyyy8VMoBLyrZt23D//n38+++/MDQ0\npCXVRUVF8PLykoqkWlJEIhGysrJQUFAAOzs7CIVCxMfHQ01NDRwORyHLadRyla6uLpycnOpcbYtf\nmVMCA6pqXlYOBuIFk717926WzJwazKnGcFFRUa3etKdaUGRlZTWrg2tTtG/fHubm5uBwOMjLy8Oj\nR48ataShqKmpwc8//4zIyEgcPnwYPXv2lEosLWXx4sVYt24dWCwW3Yb64MGDCo3pfYPZwwFw4cIF\n7NmzBxEREU0e29ZdXHk8XoODlrQk1ZLSmCiAkv6mpaW1SNnWUsSlzk25KItTXV2NjIwMlJSUtNgV\noCFkYUvTVGO4pqiursbLly+hpqYm8zofSuGYkZEBkUgEY2PjOn2PHj9+DE9PT4wbNw6rVq2Si+rt\nbRrzQXvXY/fu3YO/vz/++ecfAKAvYtesWSP7gBUHIxpoDlOmTMGwYcMwZ86ceo/l5eXR1dPR0dGY\nNGkSLQN+H2mppFpSmiMK4HK5SE9PR3V1NYyNjaXmBfY2VEFpaxR6tbW1yMrKQl5eXqvVYrK2pXm7\nMRzVPbWx900VAWdnZ8Pa2rpBpaMsqaysREZGBrZt2wYLCwvw+XzcuHEDISEhsLe3l2ss4rydVHJz\nc2lRyS+//IKoqCicOnWqzjkCgQBWVla4du0a2Gw2nJ2dceLECfTo0UPu8csRJuFISmVlJYyNjZGa\nmkrbluzbtw8AsGjRIuzevRvBwcFQUVFBx44dsX37dvTv31+RIUsFSSXVzaGlTgHiswhp+qDx+XzE\nx8dDSUmJrhlpLSKRCLm5uS1SiynClqasrKxOH5m3izMp9wJNTU2Ym5srtJbl1q1b2LhxI1JSUjBz\n5kx4eXmBzWYrJJaGbGkiIyMRFxdHz8z3798PfX39OrY0ABAWFgZvb28IhUJ4eHi0dVsaacAkHAbJ\n4PF4OHLkCPbt29dil+rq6mokJCS02imAkinn5+e3ai9CvBW2rAZ2qnNmeno6lJWV6aXBxlC0LQ2P\nx6vTGM7AwAA5OTltwr2gtrYWv/zyCy5fvox9+/ahZ8+e+PPPP7F79278+uuvcHR0VFhsDBLBJByG\n5tESl2pZOQVQvU6ysrLwySef1DGKbIrKyso6V+zyGNjFi17fXhokhLQpWxqBQIBXr17RQpiePXsq\nRAJO8eLFCyxbtgxDhw7F999/X+dihRqb3tfl648IJuEoCg6HA01NTSgrK0NFRaVeRz9CCLy8vBAW\nFgY1NTUcPnwYDg4OCoq2PoQQXLt2DUFBQe90qS4rK0NCQgK6dOkCDocjk6UYamM9PT39naaWwH+9\nfAoLC2FtbS3XhmwUbwsMtLS0kJiYiE8++QRmZmYKt6URiUTIyMhAfn4+rK2twefzkZGRgfbt29P7\nPPKitrYWO3fuRGhoKIKDg5lZzPsNk3AURVMqtrCwMOzatQthYWGIioqCl5cXoqKi5BylZDQkqS4p\nKUFUVBS6du0qN6cAcVNLAPWKHRuqzFckfD4fT58+RVlZGfT19WFhYaHwrqkVFRV4+fIlrRoU/x1J\nuzFcU7x8+RLLly+Hm5sb1q9fLxMboYasaXx9fXHp0iWoqqrC3Nwchw4danDG2dRFI0M9mISjKJpK\nOAsXLsTAgQPpIlNra2tERka2CUuVxkhMTMTPP/+Mu3fvoqqqCitWrICHh4dCljooZVtVVRUMDQ1R\nVlaG6upq2NjYNLv9gCwQT36GhoYoKChAZmam3O1oKMRnfra2to3OEAHpNIZ7FwKBALt378bZs2cR\nHBwsk/5OFA1Z00RERGDQoEFQUVHBqlWrAABBQUH1zm3rpQ9tEIkHAsZpQMqwWCwMGTIEjo6OCAkJ\nqff4+1iJ3KFDB7x+/Ro9e/bE2LFj8fvvv+PAgQOorq6Weyyampro2bMnDAwMkJiYiMLCQujo6Cjc\nbJMy/0xJSYGdnR04HA5UVFRgYGCAvn37QldXF/Hx8YiLi0NJSQmaeaHXIrhcLmJiYsBiseDk5PTO\nZAMAampqsLGxgaOjI4RCIaKjo5GcnAw+n9/qWBISEjBy5EiUl5fj9u3bMk02wBtrmreLVocOHUrv\n6bm4uCArK0umMTDUh3EakDK3b98Gm81GQUEB3N3dYWNjA1dXV0WH1Sq2bdsGLy8vDBo0CMB/kmo3\nNzepSaolRbyGpX///mCxWLRnm56eHoyMjOReJFhUVISkpCQYGho26BFH2dHo6OjQAgOqBYIsao8o\n656SkhJ079692cue7dq1A4fDgbGxMfLz8/H48eMm99AaQyAQYO/evThz5gz27t2Lfv36Net8WXHw\n4ME6TvHiUBeNysrKWLhwIRYsWCDn6D5cmIQjZai6AV1dXYwfPx7R0dF1Eg6bzUZmZiZ9OysrS2G1\nBpKyc+fOOrc7deqENWvW4Ntvv8WRI0cwevToFkuqJUW8ONHCwqLOcoepqSmMjY2Rm5uL2NjYZivb\nWkpLbGm0tLRgZ2dHCwxevXol1eUrqkupnp4enJycWpXMlJSUoK+vj27duqGkpATJyckQiUQwMTFB\nly5dmnzupKQkLF++HP369cOdO3ek2h22NWzatAkqKiqYPn16g4+35qIxOzubLnoG3nxuGZXdfzBL\nalKksrISXC6X/jkiIqKe/9PYsWNx9OhREEJw//59aGtrt+n9m3fRoUMHLFy4EDExMejfvz9mzJiB\nJUuW0H1lpEVFRQViY2NRXV3dqJGksrIyDA0N0a9fP3Tq1AnPnj3D06dP6b+HNKF6w1Dr/L169Wr2\nYNqxY0dYW1tLbflKKBQiKSkJSUlJ6NmzJzgcjtQGOhaLhc6dO6NPnz6wsbFBQUEBoqKikJWVBaFQ\n2GAse/fuxZw5cxAQEIAtW7
```

Comment:
- Refresh button does not seem to do anything (automatic update)
- dx, dy, dz, x0, y0, elevation widget are  out of range for the figure plot (widget's ranges are like up to a 1000m gure range is about 50m, making it actually hard to use the widget but by typing a value. The slider goes out of bounds very quickly)